pkg/workload/histogram/exporter/exporter_test.go

```diff
@@ -209,7 +209,7 @@ func TestOpenMetricsFileWithJson(t *testing.T) {
 							return errors.Errorf("invalid summary count: %f", countValue)
 						}
 					}
-
+					quantile *= 100
```

Comment:
Can you please add a comment here as well mentioning why we have done *100?

Reply:
Done